spec/lib/whitehall_importer/create_file_attachment_revision_spec.rb

```diff
@@ -32,5 +32,18 @@
         "File attachment does not exist: #{whitehall_file_attachment['url']}",
       )
     end
+
+    it "for a file attachment with requirements issues" do
+      too_long_title = ("A" * Requirements::FileAttachmentChecker::TITLE_MAX_LENGTH + "A")
```

Comment:
Think this is a little easier to read:

```suggestion
      too_long_title = (Requirements::FileAttachmentChecker::TITLE_MAX_LENGTH * "A") + "A"
```

Reply:
Hmm ruby didn't like that approach and threw and error converting a string to fixnum so I went with `("A" * Requirements::FileAttachmentChecker::TITLE_MAX_LENGTH) + "A"`, don't quite understand why ruby accepted that way round of doing it! 🤔